content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -45,67 +115,60 @@ team. These credentials are only used to generate the
 [authorization](#authorization) URL DVC will later prompt to visit in order to
 connect to the Google Drive.
 
-> Note that Google Drive API has usage limits/quotas per credentials in a
-> _project_ (which you can review in the
+> Note that the Google Drive API has usage limits/quotas per _project_ client
+> (which you can review in the
 > [OAuth consent screen](https://console.developers.google.com/apis/credentials/consent)).
 > Please keep this in mind when sharing them, or you may
 > [exceed the limits](https://developers.google.com/drive/api/v2/handle-errors?hl=ro#resolve_a_403_error_usage_limit_exceeded).
 
-## URL format for Google Drive remotes
-
-A GDrive DVC remote URLs is needed for the `dvc remote add` command. It can be
-constructed with a _base_, and an **optional** _path_ to a folder i.e.
-`gdrive://<base>/path/to/folder`. The base can be one of:
-
-1. `root` - indicates your topmost Google Drive directory.
+### Enabling your GC project with OAuth credentials
```

Comment:
I don't quite understand this title ... why do we enable GC project?

Reply:
make it - `Configuring DVC to use you custom credentials`?

Reply:
Yeah. How about we just remove the header actually? I feel like it's simply part of the custom gc project section anyway. See f6b1aebc.

Reply:
p.s. the intro is kinda long now...